app/views/ruling_information.scala.html

```diff
@@ -0,0 +1,104 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.FrontendAppConfig
+@import utils.Dates
+
+@(appConfig: FrontendAppConfig, c: Case)(implicit request: Request[_], messages: Messages)
+
+    @main_template(
+        title = s"${messages("rulingInformation.title")}",
+        appConfig = appConfig,
+        bodyClasses = None)  {
+
+        @components.back_link()
+
+        @components.heading(s"${messages("rulingInformation.Heading")} GB${c.reference}")
+
+        @components.subheading(messages("rulingInformation.Summary"))
+
+
+        <div class="grid-row">
+            <div class="grid-full-width">
+                <div class="column-one-third">@messages("rulingInformation.Holder")</div>
+                <div class="column-two-thirds">@c.application.holder.businessName</div>
+            </div>
+        </div>
+        </br>
+        <div class="grid-row">
+            <div class="grid-full-width">
+                <div class="column-one-third">@messages("rulingInformation.Ruling")</div>
+                <div class="column-two-thirds">GB@c.reference</div>
```

Comment:
Similar here

Reply:
After post-amigos this has been removed now, so no longer a thing